core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -265,23 +292,84 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       threadId);
     const threadMessageBackendDicts = response.messages;
     this.reviewMessage = threadMessageBackendDicts.map(
-      m => ThreadMessage.createFromBackendDict(m))[1].text;
+      m => ThreadMessage.createFromBackendDict(m))[1]?.text ?? '';
```

Comment:
This code is a bit complicated, can you rewrite it into a non-shortened version.

Reply:
Done